Exiled.Events/Handlers/Item.cs

```diff
@@ -21,32 +21,32 @@ public static class Item
         /// <summary>
         ///     Invoked before the ammo of an firearm are changed.
         /// </summary>
-        public static event CustomEventHandler<ChangingAmmoEventArgs> ChangingAmmo;
+        public static Event<ChangingAmmoEventArgs> ChangingAmmo { get; set; } = new ();
 
         /// <summary>
         ///     Invoked before item attachments are changed.
         /// </summary>
-        public static event CustomEventHandler<ChangingAttachmentsEventArgs> ChangingAttachments;
+        public static Event<ChangingAttachmentsEventArgs> ChangingAttachments { get; set; } = new();
 
         /// <summary>
         ///     Invoked before receiving a preference.
         /// </summary>
-        public static event CustomEventHandler<ReceivingPreferenceEventArgs> ReceivingPreference;
+        public static Event<ReceivingPreferenceEventArgs> ReceivingPreference { get; set; } = new();
```

Comment:
Isnt a breaking changes?

Reply:
I know, you added overload for + but anyways

Reply:
I don't care about breaking change